### ISSUE TITLE
My score not displayed

### ISSUE THREAD

**Issue by abchugh** [slatex/sTeX-React#461]
A content ERROR was logged by "a user" at the following url:

https://courses.voll-ki.fau.de/quiz-dash/ai-1

## The issue as described by the user:
```
My Score values are not displayed
```

## The text highlighted while reporting this issue:
```
Score
```

**Comment by kohlhase** [comment]
Am I understanding this correctly you want to see the partial score while you are taking the quiz? 
Why would you even consider this  and waste time checking the score? 
Showing the score during the quiz would be counter to the result release time configuration. 
I am sure that I am misunderstanding something.

**Comment by abchugh** [comment]
No, this is not the current partial score but that of previous quizzes. The query to get the previous quiz scores contributes to the database load so I have disabled them for now.

**Comment by kohlhase** [comment]
Ah, thanks.
